app/util/ContentUpgrade.scala

```diff
@@ -45,45 +44,34 @@ object ContentUpgrade {
   }
 
   def upgradeItem(json: JValue): JValue = {
-    Try({
-      val jsonString = JsonMethods.compact(JsonMethods.render(json))
-      val maybeParsedJson: Option[Json] = parser.parse(jsonString).right.toOption
-      val maybeCapiContent: Option[Content] = maybeParsedJson.flatMap(json => json.as[Content].right.toOption)
-      (json, maybeCapiContent) match {
-        case (jsObject: JObject, Some(content)) =>
-          val cardStyle = CardStyle(content, TrailMetaData.empty)
-          val initialMetaDataMap: Map[String, Boolean] = ResolvedMetaData.toMap(ResolvedMetaData.fromContent(content, cardStyle))
+    Try(getUpgradedItem(json)) match {
+      case Success(capiItem) => capiItem
+      case Failure(_) => json
+    }
+  }
 
-          // ResolvedMetaData here has a ton of booleans, but some may have been set naively
-          // If the cutout bool is set, we want to find the cutout values and add them in to a sub-object
-          val (metaDataMap, maybeFirstContributorWithCutout) = initialMetaDataMap.get("imageCutoutReplace") match {
-            case Some(true) => {
-              getBylineLargeImageUrlOption(content)  match {
-                case bylineLargeImageUrlOption@Some(_) => (initialMetaDataMap, bylineLargeImageUrlOption)
-                // If we can't find a cutout value, turn off the cutout bool
-                case None => (initialMetaDataMap + ("imageCutoutReplace" -> false), None)
-              }
-            }
-            case _ => (initialMetaDataMap, None)
-          }
+  private def getUpgradedItem(json: JValue): JValue = {
+    val jsonString = JsonMethods.compact(JsonMethods.render(json))
+    val maybeParsedJson: Option[Json] = parser.parse(jsonString).right.toOption
+    val maybeCapiContent: Option[Content] = maybeParsedJson.flatMap(json => json.as[Content].right.toOption)
+    (json, maybeCapiContent) match {
+      case (jsObject: JObject, Some(content)) => {
 
-          jsObject ~ ("frontsMeta" ->
-            ("defaults" -> metaDataMap)
-              ~ ("tone" -> cardStyle.toneString)
-              ~ ("cutout" -> maybeFirstContributorWithCutout)
+        val prefill = CapiPrefiller.prefill(content)
+
+        (jsObject
+          ~ ("frontsMeta" ->
+              ("defaults" -> ResolvedMetaData.toMap(prefill.metaData))
+              ~ ("tone" -> prefill.tone)
+              ~ ("cutout" -> prefill.cutout.map(_.src))
+              ~ ("pickedKicker" -> prefill.pickedKicker)
+              ~ ("mediaType" -> prefill.mediaType.map(_.toString))
+            )
           )
-        case _ => json
       }
-    }) match {
-      case Success(capiItem) => capiItem
-      case Failure(_) => json
+      case _ => json
```

Comment:
Again, should we log a warning here? This is an unexpected fallback IIUC?

Reply:
Done